apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/PatientResourceProvider.java

```diff
@@ -29,6 +29,7 @@
 import gov.cms.bfd.server.war.commons.QueryUtils;
 import gov.cms.bfd.server.war.commons.TransformerConstants;
 import java.math.BigDecimal;
+import java.time.LocalDateTime;
```

Comment:
(Just hanging this comment here for lack of anywhere better.)

I think we want to end up with a FHIR ["composite" search](https://www.hl7.org/fhir/search.html#composite) that looks something like this?

```
GET [base]/Coverage?ptdcntrct=eq2020-08$http://something.cms.gov/ptdcntrct|<whatever_the_contract_number_is>
```